.vscode/settings.json

```diff
@@ -0,0 +1,6 @@
+{
+    "cSpell.enabled": true,
+    "cSpell.enableFiletypes": [
+        "raml"
+    ]
+}
```

Comment:
Missing newline?

Reply:
Drat. This shouldn't have been committed. Removed in latest commit and added to gitignore.